src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, sans-serif;
+}
+
+nav {
+  max-width: 1200px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 0 auto;
+  padding: 0 20px;
+  box-sizing: border-box;
+}
+
+.nav__list {
+  list-style: none;
+  gap: 20px;
+  display: flex;
+}
+
+img {
+  margin-left: 30px;
+  width: 40px;
+  height: 40px;
+}
+
+.link,
+.blue_link {
+  padding: 5px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+}
+
+.blue_link {
+  color: #00acdc;
+}
+
+.rectangle {
+  position: absolute;
+  border-radius: 8px;
+  margin-top: 15px;
+  width: 37px;
+  height: 4px;
+  background: #00acdc;
+}
+
+a:hover {
+  text-decoration-line: underline;
+  text-decoration-color: #00acdc;
+}
+
+@media (max-width: 1200px) {
```

Comment:
Remove the '@media (max-width: 1200px)' query as the layout should fit on both 1024px and 1200px screens without any extra styles.